projects/1.16/assets/portable-stonecutter/portable_stonecutter/lang/zh_cn.json

```
@@ -1 +1,7 @@
-{}
\ No newline at end of file
+{
+    "item.portable_stonecutter.portable_stonecutter": "便携式切石机",
+    "itemGroup.portableStonecutter": "便携式切石机",
+    "jei.portable_stonecutter:jei_anvil_flattening": "铁砧压平",
+    "info.portable_stonecutter:portable_stonecutter": "将原料作为物品扔在地上，然后在上面放一个铁砧来制作。",

```

Throw the ingredients as items on the ground and drop an anvil on them to craft.
```suggestion
    "info.portable_stonecutter:portable_stonecutter": "把原料丢在地上，然后让一个铁砧在它上方落下，制作就完成了。",
```

---

```suggestion
    "info.portable_stonecutter:portable_stonecutter": "把原料物品丢在地上，然后在它上方放置一个铁砧，铁砧砸下来后制作就完成了。",
```